packages/builtins/src/collections/user/index.ts

```diff
@@ -32,11 +36,13 @@ const exposedFunctions: Record<keyof typeof functions, AccessCondition> = {
   removeFile: true,
   insert: true,
   authenticate: 'unauthenticated',
-  activate: 'unauthenticated',
+  activate: ['unauthenticated', 'root'],
```

Comment:
This isn't needed as `'unauthenticated'` covers both unauthenticated and authenticated users from all roles.